packages/cactus-test-plugin-htlc-eth-besu-erc20/src/test/typescript/integration/plugin-htlc-eth-besu-erc20/withdraw-endpoint.test.ts

```diff
@@ -58,463 +59,234 @@ const web3SigningCredential: Web3SigningCredential = {
 } as Web3SigningCredential;
 
 const testCase = "Test withdraw endpoint";
-
-test("BEFORE " + testCase, async (t: Test) => {
-  const pruning = pruneDockerAllIfGithubAction({ logLevel });
-  await t.doesNotReject(pruning, "Pruning did not throw OK");
-  t.end();
-});
-
-test(testCase, async (t: Test) => {
-  t.comment("Starting Besu Test Ledger");
-  const besuTestLedger = new BesuTestLedger({
-    logLevel,
-    envVars: [...BESU_TEST_LEDGER_DEFAULT_OPTIONS.envVars, "BESU_LOGGING=ALL"],
-  });
-  await besuTestLedger.start();
-
-  test.onFinish(async () => {
+describe(testCase, () => {
```

Comment:
@awadhana If this is one of those test cases that you broke into two as part of the Jest migration then please also add in the other test file as part of the PR (if not already merged in another PR). Otherwise we would leave certain coverage missing if we merged this as-is.

Reply:
Alright, I will be sure to do that once I get the other have passing successfully. Thank you!

Reply:
@awadhana My pleasure and thank you very much!

Reply:
@petermetz , this is the other half from what we mentioned above! 

https://github.com/hyperledger/cactus/pull/1889

Reply:
@awadhana Got it, thank you!